### PR TITLE
an/SKY-9: Login page design

### PR DESCRIPTION
**Pull request by anurmamedov**
* login details upload and readme text updated

1.fontawesome added and placeholders
2.logo size adjusted with the rest
3.sign in button size increased

sign size   logo rounded register design detail

large size login design
medium size login design
small size login design

### CODE REVIEW

**Review by Aymammet — CHANGES_REQUESTED** [review]
1. Show error messages how they look like under each input. (maybe red color)
a. your password is wrong
b. this username does not exists

2. label should be on top of the input

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# Sky: Social meida application named Sky
+# SKY: Social Media Application, named Sky
```
Comment (Aymammet) [inline README.md:1]: remove

File: templates/login.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        </div>
+        <form id="login" class="container col-md-5 col-lg-4 col-xl-3 col-sm-6 mx-auto mt-5">
+            <!-- Email input -->
+            <div class="form-outline mb-4 ">
+              <input type="email" id="form2Example1" class="form-control"/>
+              <label class="form-label" for="form2Example1">Email address</label>
+            </div>
+            <!-- Password input -->
+            <div class="form-outline mb-4">
+              <input type="password" id="form2Example2" class="form-control" />
```
Comment (Aymammet) [inline templates/login.html:13]: placeholder and fontawesome icon

**Review by Aymammet — CHANGES_REQUESTED** [review]

File: templates/login.html (hunk 2 above)
Comment (Aymammet) [reply templates/login.html]: is this done?

Comment (anurmamedov) [reply templates/login.html]: yes, it is

**Review by Aymammet — APPROVED** [review]

**Pull request merged**